lib/roo_on_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module RooOnRails
-  VERSION = '2.0.2'.freeze
+  VERSION = '2.1.0'.freeze
```

Comment:
minor bump for major support for a rails version 🤔

Reply:
I think this is OK since the gem is still backwards compatible and hasn't been following Rails versions I am afraid.